builder/oracle/classic/step_snapshot.go

```diff
@@ -26,16 +27,18 @@ func (s *stepSnapshot) Run(_ context.Context, state multistep.StateBag) multiste
 	snapshotInput := &compute.CreateSnapshotInput{
 		Instance:     fmt.Sprintf("%s/%s", config.ImageName, instanceID),
 		MachineImage: config.ImageName,
+		Timeout:      time.Minute * 20,
```

Comment:
any chance this would be longer? Do we want to let this be configurable? too bad it doesn't use contexts...

Reply:
I've already opened an issue to make this configurable.